README.md

```diff
@@ -16,3 +16,18 @@ make test
 ```
 make format
 ```
+
+
+## install talib in ubuntu
+
+These command may work
+```
+apt update && apt install build-essential wget -y
+wget https://artiya4u.keybase.pub/TA-lib/ta-lib-0.4.0-src.tar.gz
+tar -xvf ta-lib-0.4.0-src.tar.gz
+
+cd ta-lib/ &&./configure --prefix=/usr && make && make install
+rm -rf ta-lib-0.4.0-src.tar.gz
+pip install TA-Lib
+```
```

Comment:
these diff are from other pr, if you do `git pull origin master` you may be able to get a clean pr without this part.